### ISSUE TITLE
Add basic support for alternative storage engines (default is 'sparql')

### ISSUE THREAD

**Issue by nevali** [bbcarchdev/twine#20]
Add a 'null' storage engine which no-ops encapsulated writes (i.e., graph PUTs), but still allows SPARQL queries if a source has been configured.

**Comment by nevali** [comment]
Depends upon #23

**Comment by nevali** [comment]
This is no longer needed; a workflow can be configured which omits `sparql-put`
